docs/devguide/debugging_guide.rst

```diff
@@ -121,3 +121,33 @@ like that - it’ll expand the time. There’s only so much we can do to work ar
 that while the relevant computer things things are “UP and OK” and has started a TFTP/PXE boot process, the
 switch hasn’t brought the NIC link up. So we added an explicit sleep in there in the monorail.ipxe to extend
 'the time to let networks converge so that the process has a better chance of succeeding.
+
+
+*Question*:
+
+what is this error indicate?
+
+.. code::
+
+    E 2016-03-24T23:04:00.548Z [on-dhcp-proxy] [Services.ErrorPublisher] [Server] Event Loop Blocked
```

Comment:
Minor suggestion, not blocking since it doesn't matter the debugging guide . In the latest code, the first letter `E` has been changed to use the full log level name `error`